### DISCUSSION TITLE
The DataGridCheckBoxColumn of the DataGrid requires two clicks to be selected

### DISCUSSION THREAD

**Discussion by qfmee** [AvaloniaCommunity/Material.Avalonia#319] [General]
avalonia:11.0.5
windows 11
The DataGridCheckBoxColumn of the DataGrid requires two clicks to be selected. The first click selects the row, and the second click selects the CheckBox！Usually, we should select CheckBox after clicking once! 
Is there any good way to solve it？

**Comment by SKProCH** [comment]
Hello, @qfmee 

Seems like this is default Avalonia behavior. You probably should address it to Avalonia repo.
